integration-cli/docker_cli_events_test.go

```diff
@@ -230,6 +230,30 @@ func (s *DockerSuite) TestEventsImageUntagDelete(c *check.C) {
 	}
 }
 
+func (s *DockerSuite) TestEventsImageTag(c *check.C) {
+	time.Sleep(time.Second * 2) // because API has seconds granularity
```

Comment:
Events storage has only seconds granularity. If we don't sleep earlier events from previous test cases show up here too.